### ISSUE TITLE
Illegal argument exception for boolean type

### ISSUE THREAD

**Issue by ThaBalla1148** [StrongJoshua/libgdx-inGameConsole#18]
Creating a command with a boolean as an argument does not work.

Example:
```java
public class MyCommandExecutor extends CommandExecutor {
		public void test(boolean t) {
			console.log(String.valueOf(t));
		}
	}
```

**Comment by ThaBalla1148** [comment]
This part of exeCOmmand(String command) does not account for a boolean:
```java
// attempt to convert arguments to numbers. If the conversion does not work, keep the argument as a string.
		Object[] args = null;
		if (sArgs != null) {
			args = new Object[sArgs.length];
			for (int i = 0; i < sArgs.length; i++) {
				String s = sArgs[i];
				try {
					int j = Integer.parseInt(s);
					args[i] = j;
				} catch (NumberFormatException e) {
					try {
						float f = Float.parseFloat(s);
						args[i] = f;
					} catch (NumberFormatException e2) {
						args[i] = s;
					}
				}
			}
		}
```

I'm actually looking into rewriting and fixing this code.

**Comment by ThaBalla1148** [comment]
Fixed in pull request #19
